packages/components/src/font-size-picker/index.js

```diff
@@ -118,6 +148,16 @@ export default function FontSizePicker( {
 					{ __( 'Reset' ) }
 				</Button>
 			</div>
+			<div>
+				<ToggleControl
+					label="Advanced font sizing"
+					onChange={ () =>
+						setAdvancedFontSizing( ! advancedFontSizing )
+					}
+					help="Toggle to activate custom font-sizing settings"
```

Comment:
Should be translatable

```suggestion
					help={ __( 'Toggle to activate custom font-sizing settings' ) }
```